RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
 [internal] Fixed an issue where source and platform tags were not added to a Zendesk ticket if the account has no blogs. [#17084]
 * [*] Set the post formats to have 'Standard' first and then alphabetized the remaining items. [#17074]
 * [*] Added pull-to-refresh to My Site.
+* [***] Weekly Roundup is live for all users!
```

Comment:
```suggestion
* [*] Added pull-to-refresh to My Site. [#17089]
* [***] Weekly Roundup: users will receive a weekly notification that presents a summary of the activity on their most used sites [#17066, #17116]
```

Reply:
I ended up adding this when @Gio2018 mentioned it to me via DM... so we can ignore this now.